CircleciScripts/ReleaseConfiguration.json

```diff
@@ -14,7 +14,6 @@
 			"aws-android-sdk-auth-userpools",
 			"aws-android-sdk-autoscaling",
 			"aws-android-sdk-cloudwatch",
-			"aws-android-sdk-cognito",
 		    // "aws-android-sdk-cognitoauth",
```

Comment:
Hm, this doesn't look good. `//` isn't valid in the middle of a JSON document.

Reply:
hmm, it was always there.. that _is_ strange though. You are right that JSON doc doesn't support this on hindsight, but I didn't see anything wrong with it until you just mentioned haha